sdk/python/feast/infra/offline_stores/redshift.py

```diff
@@ -478,19 +479,23 @@ def _upload_entity_df_and_get_entity_schema(
 */
 {{ featureview.name }}__latest AS (
     SELECT
-        {{featureview.name}}__entity_row_unique_id,
```

Comment:
This should have fail with the current unit tests ...

Reply:
Think the tests you added might have been accidentally removed... Mind adding them back to the universal_historical_retrieval test?